README.md

```diff
@@ -49,14 +49,14 @@ The `server` command supports the following additional arguments.
 
 ```shell
 OPTIONS:
-   --addr value           The address to listen on for probe messages (default: ":8123") [$ADDR]
-   --buffer-size value    The size of the read buffer used by the server (default: 512) [$BUFFER_SIZE]
-   --read-timeout value   The duration after which the server should timeout when reading from a connection (default: 2s) [$READ_TIMEOUT]
-   --write-timeout value  The duration after which the server should timeout when writing to a connection (default: 5s) [$WRITE_TIMEOUT]
-   --log.format value     Specify the format of logs. Supported formats: 'logfmt', 'json', 'console' [$LOG_FORMAT]
-   --log.level value      Specify the log level. e.g. 'debug', 'info', 'error'. (default: "info") [$LOG_LEVEL]
-   --log.ctx value        A list of context field appended to every log. Format: key=value. [$LOG_CTX]
-   --help, -h             show help (default: false)
+   --addr value                         The address to listen on for probe messages (default: ":8123") [$ADDR]
+   --buffer-size value                  The size of the read buffer used by the server (default: 512) [$BUFFER_SIZE]
+   --read-timeout value                 The duration after which the server should timeout when reading from a connection (default: 2s) [$READ_TIMEOUT]
+   --write-timeout value                The duration after which the server should timeout when writing to a connection (default: 5s) [$WRITE_TIMEOUT]
+   --log.format value                   Specify the format of logs. Supported formats: 'logfmt', 'json', 'console' [$LOG_FORMAT]
+   --log.level value                    Specify the log level. e.g. 'debug', 'info', 'error'. (default: "info") [$LOG_LEVEL]
+   --log.ctx value [ --log.ctx value ]  A list of context field appended to every log. Format: key=value. [$LOG_CTX]
```

Comment:
Why did this change? An update to urfave/cli or something?

Reply:
I think so, or it has been cropped manually. In any case this output can also be produced with running `--help` on the `main` branch. I have just updated it to make sure we're up2date.